server/models/User.test.js

```diff
@@ -1,10 +1,11 @@
 /* eslint-disable flowtype/require-valid-file-annotation */
-import { flushdb, seed } from '../test/support';
+import { flushdb } from '../test/support';
+import { buildUser } from '../test/factories';
 
 beforeEach(flushdb);
 
 it('should set JWT secret and password digest', async () => {
-  const { user } = await seed();
+  const user = await buildUser({ password: 'test123!' });
```

Comment:
We should gradually start replacing the `seed` calls with individual factory calls to make the tests run faster.